isis/tests/FunctionalTestsCaminfo.cpp

```diff
@@ -0,0 +1,313 @@
+    EXPECT_NEAR(csvLine[46].toDouble(), 255.64554860056, 0.000001);
+    EXPECT_NEAR(csvLine[47].toDouble(), 10.086794148631, 0.000001);
+    EXPECT_NEAR(csvLine[48].toDouble(), 255.96651410281, 0.000001);
+    EXPECT_NEAR(csvLine[49].toDouble(), 9.928647808629, 0.000001);
+    EXPECT_NEAR(csvLine[50].toDouble(), 256.14606965798, 0.000001);
+    EXPECT_NEAR(csvLine[51].toDouble(), 10.279980555851, 0.000001);
+    EXPECT_NEAR(csvLine[52].toDouble(), 255.82316032959, 0.000001);
+    EXPECT_NEAR(csvLine[53].toDouble(), 10.434709827388, 0.000001);
+    EXPECT_NEAR(csvLine[54].toDouble(), 80.528382053153, 0.000001);
+    EXPECT_NEAR(csvLine[55].toDouble(), 12.13356433166, 0.000001);
+    EXPECT_NEAR(csvLine[56].toDouble(), 70.127983086993, 0.000001);
+    EXPECT_NEAR(csvLine[57].toDouble(), 332.65918485196, 0.000001);
+    EXPECT_NEAR(csvLine[58].toDouble(), 9.9273765164008, 0.000001);
+    EXPECT_NEAR(csvLine[59].toDouble(), 294.73518831328, 0.000001);
+    EXPECT_NEAR(csvLine[60].toDouble(), 7.7862975334032, 0.000001);
+    EXPECT_NEAR(csvLine[61].toDouble(), 4160.7294345949, 0.000001);
+    EXPECT_NEAR(csvLine[62].toDouble(), 762.37204489156, 0.000001);
+    EXPECT_NEAR(csvLine[63].toDouble(), 18.904248476287, 0.000001);
+    EXPECT_NEAR(csvLine[64].toDouble(), 18.904248476287, 0.000001);
+    EXPECT_NEAR(csvLine[65].toDouble(), 18.904248476287, 0.000001);
+    EXPECT_NEAR(csvLine[66].toDouble(), 18.913336801664, 0.000001);
+    EXPECT_NEAR(csvLine[67].toDouble(), 92.033828011827, 0.000001);
+    EXPECT_NEAR(csvLine[68].toDouble(), 118.87356332432, 0.000001);
+    EXPECT_NEAR(csvLine[69].toDouble(), -22.740326163641, 0.000001);
+    EXPECT_NEAR(csvLine[70].toDouble(), 319.09846558533, 0.000001);
+    EXPECT_NEAR(csvLine[71].toDouble(), 240.08514371127, 0.000001);
+    EXPECT_NEAR(csvLine[72].toDouble(), 267.53187323573, 0.000001);
+    EXPECT_NEAR(csvLine[73].toDouble(), 10.078847382918, 0.000001);
+    EXPECT_NEAR(csvLine[74].toDouble(), 253.65422317887, 0.000001);
+    EXPECT_NEAR(csvLine[75].toDouble(), 0.0092584293412006, 0.000001);
+    EXPECT_NEAR(csvLine[76].toDouble(), -0.21479478952768, 0.000001);
+    EXPECT_NEAR(csvLine[77].toDouble(), 1.3359751259293, 0.000001);
+    EXPECT_NEAR(csvLine[78].toDouble(), 2.4227562244446, 0.000001);
+    ASSERT_EQ(csvLine[79].toStdString(), "FALSE");
+    ASSERT_EQ(csvLine[80].toStdString(), "FALSE");
+    ASSERT_EQ(csvLine[81].toStdString(), "FALSE");
+    EXPECT_NEAR(csvLine[82].toDouble(), 19.336214228383, 0.000001);
+    EXPECT_NEAR(csvLine[83].toDouble(), 19.336214228383, 0.000001);
+    EXPECT_NEAR(csvLine[84].toDouble(), 19.336214228383, 0.000001);
+    EXPECT_NEAR(csvLine[85].toDouble(), 19.336214228383, 0.000001);
+}
+
+
+TEST_F(DefaultCube, FunctionalTestCaminfoDefault) {
+    QString outFileName = tempDir.path() + "/outTemp.csv";
+    QVector<QString> args = {"to="+outFileName,
+        "ISISLABEL=true", "ORIGINAL=true", "STATISTICS=true", "CAMSTATS=true",
```

Comment:
Should we be testing the default parameters here instead of setting them, unless it is crucial to get good data, like setting polylinc and polysinc?

Reply:
I think these ones need to be there. I was looking at the documentation page for caminfo, and it looks like the default value for the parameters listed is false.